src/flows/shared/components/WizardFooter/WizardFooter.tsx

```diff
@@ -58,6 +62,11 @@ function WizardFooter({
         setIsLoading(false);
       }
 
+      if (isFromGalleryPage) {
+        back();
```

Comment:
Need to use `back` instead of `push` to maintain scroll position when the user finished edit the collection

Reply:
great catch 🔥